src/agent/codecoveragepublisher.ts

```diff
@@ -0,0 +1,233 @@
+import testifm = require('vso-node-api/interfaces/TestInterfaces');
+import ctxm = require('./context');
+import cm = require('./common');
+import utilities = require('./utilities');
```

Comment:
Common agent implementation is only possible with core clr agent. Until then we cannot have a common agent implementation.